src/plugins/nomics.js

```diff
@@ -0,0 +1,49 @@
+// @flow
+
+import {
+  type EdgeCorePluginOptions,
+  type EdgeRatePlugin
+} from 'edge-core-js/types'
+
+export function makeNomicsPlugin (opts: EdgeCorePluginOptions): EdgeRatePlugin {
+  const { io, initOptions } = opts
+  const { apiKey } = initOptions
+
+  if (apiKey == null) {
+    throw new Error('No Nomics exchange rates API key provided')
+  }
+  return {
+    rateInfo: {
+      displayName: 'Nomics'
+    },
+
+    async fetchRates (pairsHint) {
+      console.log(
```

Comment:
If you use `io.console.log` instead of `console.log`, it will go over the bridge to the main app. Not a dealbreaker, since we don't necessarily need these logs.